scripts/generateChangedIconsCommentMarkup.mjs

```diff
@@ -26,7 +26,7 @@ const getImageTagsByFiles = (files, getBaseUrl, width) =>
 
       return `<img title="${file}" alt="${file}" ${widthAttr} src="${url}/${base64}.svg"/>`;
     })
-    .join('');
+    .join(width.includes('%') ? ' ' : '');
```

Comment:
This will squash previews on mobile devices, why not keep them at 400px, but still add the space?